packages/core/src/components/menu/menu.tsx

```diff
@@ -0,0 +1,586 @@
+    providerName: 'MenuProvider',
+});
+
+/* -------------------------------------------------------------------------------------------------
+ * Menu.Root
+ * -----------------------------------------------------------------------------------------------*/
+
+type RootPrimitiveProps = ComponentPropsWithoutRef<typeof RadixRoot>;
+interface MenuRootProps extends RootPrimitiveProps, MenuSharedProps {}
+
+const Root = ({ ...props }: MenuRootProps) => {
+    const [sharedProps, otherProps] = createSplitProps<MenuSharedProps>()(props, [
+        'disabled',
+        'side',
+        'align',
+        'sideOffset',
+        'alignOffset',
+    ]);
+
+    return (
+        <MenuProvider value={{ dir: otherProps.dir, ...sharedProps }}>
+            <RadixRoot {...otherProps} />
+        </MenuProvider>
+    );
+};
+Root.displayName = 'Menu.Root';
+
+/* -------------------------------------------------------------------------------------------------
+ * Menu.Trigger
+ * -----------------------------------------------------------------------------------------------*/
+
+type TriggerPrimitiveProps = ComponentPropsWithoutRef<typeof RadixTrigger>;
+interface MenuTriggerProps extends TriggerPrimitiveProps {}
+
+const Trigger = forwardRef<HTMLButtonElement, MenuTriggerProps>(
+    ({ disabled, className, children, ...props }, ref) => {
+        const context = useMenuContext();
+        const isDisabled = disabled || context.disabled;
+
+        return (
+            <RadixTrigger
+                ref={ref}
+                disabled={isDisabled}
+                data-disabled={isDisabled}
+                aria-controls={undefined}
+                {...props}
+            >
+                {children}
+            </RadixTrigger>
+        );
+    },
+);
+Trigger.displayName = 'Menu.Trigger';
+
+/* -------------------------------------------------------------------------------------------------
+ * Menu.Portal
+ * -----------------------------------------------------------------------------------------------*/
+
+type PortalPrimitiveProps = ComponentPropsWithoutRef<typeof RadixPortal>;
+interface MenuPortalProps extends PortalPrimitiveProps {}
+
+const Portal = RadixPortal;
+Portal.displayName = 'Menu.Portal';
+
+/* -------------------------------------------------------------------------------------------------
+ * Menu.Content
+ * -----------------------------------------------------------------------------------------------*/
+
+type ContentPrimitiveProps = ComponentPropsWithoutRef<typeof RadixContent>;
+interface MenuContentProps extends Omit<ContentPrimitiveProps, keyof PositionerProps> {}
+
+const Content = forwardRef<HTMLDivElement, MenuContentProps>(
+    ({ className, ...props }: MenuContentProps, ref) => {
+        const context = useMenuContext();
+        const [positionerProps] = splitPositionerProps(context);
+
+        return (
+            <RadixContent
+                ref={ref}
+                className={clsx(styles.contents, className)}
```

Comment:
```suggestion
                className={clsx(styles.content, className)}
```

There is no such thing as `contents` in `css.ts`. This causes TS errors and build errors. I suggest the above modification.